container.go

```diff
@@ -707,6 +707,24 @@ func (c *Container) kill(signal syscall.Signal, all bool) error {
 	return nil
 }
 
+func (c *Container) processList(options ProcessListOptions) (ProcessList, error) {
+	state, err := c.fetchState("ps")
+	if err != nil {
+		return nil, err
+	}
+
+	if state.State != StateRunning {
+		return nil, fmt.Errorf("Container not running, impossible to list processes")
```

Comment:
Can you add `c.id` to the error here? Also, this is racy as the container might be running here, but by the time you call `processListContainer()` below it might have stopped.